docs/src/configuration/services/_index.md

```diff
@@ -1,177 +1,182 @@
+## Service timezones
 
-When defining relationships in a configuration file you will always address a service as `<servicename>`:`<endpoint>`.  See the appropriate service page for details on how to configure multiple endpoints for each service that supports it.
+All services have their system timezone set to UTC by default.
+For some services, you can change the timezone for the running service
+(this doesn't affect the container itself and so logs are still in UTC).
 
-## Connecting to a service
+* [MySQL](./mysql/_index.md#service-timezone)
+* [PostgreSQL](./postgresql.md#service-timezone)
 
-Once a service is running and exposed as a relationship, its appropriate credentials (host name, username if appropriate, etc.) will be exposed through the `PLATFORM_RELATIONSHIPS` environment variable.  The structure of each is documented on the appropriate service's page, along with sample code for how to connect to it from your application. Note that different applications manage configuration differently so the exact code will vary from one application to another.
+## Connect to a service
 
-Be aware that the keys in the `PLATFORM_RELATIONSHIPS` structure are fixed but the values they hold may change on any deployment or restart.  Never hard-code connection credentials for a service into your application.  You should re-check the environment variable every time your script or application starts.
+Once a service is running and exposed as a relationship,
+its appropriate credentials (such as the host, username, and password) are available through the `PLATFORM_RELATIONSHIPS` environment variable.
+The structure of each is documented on the appropriate service's page along with sample code for how to connect to it from your application.
 
-Access to the database or other services is only available from within the cluster.  For security reasons, they cannot be accessed directly.  However, they can be accessed over an SSH tunnel.  There are two ways to do so.  (The example here uses MariaDB but the process is largely identical for any service.)
+The keys in the `PLATFORM_RELATIONSHIPS` variable are fixed, but the values may change on deployment or restart.
+So you should check the environment variable every time your script or app starts.
 
-### Obtaining service credentials
+Access to the database or other services is only available from within the cluster.
+For security reasons, they can't be accessed directly.
 
-In either case, you will also need the service credentials.  For that, run `platform relationships`.  That will give output similar to the following:
+To connect to a service, you need the [service credentials](#obtain-service-credentials).
```

Comment:
shall we specify the specific credentials we require here?

Reply:
I'm not entirely sure what you mean. Do you think the example isn't clear enough? Would you specify it here rather than in the section below?

Reply:
Right, I get the feeling that this phrase is the intro to the 2 steps below but it's offering them the chance to jump ahead steps or have them be disconnected. 

Maybe just a 1. Obtain service credentials and 2. Connect to an SSH tunnel would address this?

Reply:
Yes, that's clearly the right approach. Just like above.

Reply:
Should be clearer now, @MariadeAnton :crossed_fingers:

Reply:
Definitely is! Thanks so much for bearing with all the little things here and there. Fantastic work all around, @CollierCZ !